rpcs3/Emu/RSX/rsx_decode.h

```diff
@@ -4526,7 +4526,7 @@ struct register_vertex_printer<u8, 4>
 		return "uchar4";
 	}
 
-	static std::string value(std::string& out, u32 v)
```

Comment:
Meaning these debugging methods are currently not used?

Reply:
It's used in VERTEX_DATA4UB.
No idea why it built.